specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2022-12-01-preview/elasticsan.json

```diff
@@ -0,0 +1,2341 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-12-01-preview",
+    "title": "ElasticSanManagement"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "ElasticSans",
+      "description": "APIs for ElasticSan operations."
+    },
+    {
+      "name": "ElasticSanOperations",
+      "description": "APIs for ElasticSan operations,"
+    },
+    {
+      "name": "VolumeGroups",
+      "description": "APIs for Volume Group operations."
+    },
+    {
+      "name": "Volumes",
+      "description": "APIs for Volume operations."
+    }
+  ],
+  "paths": {
+    "/providers/Microsoft.ElasticSan/operations": {
+      "get": {
+        "tags": [
+          "Elastic San Operations"
+        ],
+        "operationId": "Operations_List",
+        "description": "Gets a list of ElasticSan operations.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
```

Comment:
>v3 [](http://example.com/codeflow?start=69&length=2)

v5 is the current common types version (comment applies throughout new APIs).

Reply:
Hi @mentat9, previously i have tried using v5 version and it had some issues. So @TimLovellSmith suggested to switch to v3 or v4.
Ref: https://github.com/Azure/azure-rest-api-specs/pull/23012#issuecomment-1520437631

Reply:
Those issues are resolved. I won't hold signoff on this, but still recommend v5. You shouldn't see any issues: many teams are using v5 common types.